<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NativeTest</class>
 <widget class="QWidget" name="NativeTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NativeTest</string>
  </property>
  <widget class="QGroupBox" name="groupBox_Audio">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>501</width>
     <height>481</height>
    </rect>
   </property>
   <property name="title">
    <string>音频参数设置</string>
   </property>
   <widget class="QComboBox" name="cmb_recordingDevices">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_playoutDevices">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="set_recording_volume_slider">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_volume_recording">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>采集音量</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="line_recording_volume_text">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_audio_record">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>采集设备</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_playout_devices">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>播放设备</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_getmicrophonevolume">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>90</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>麦克风静音</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_volume_playout">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>播放音量</string>
    </property>
   </widget>
   <widget class="QSlider" name="set_playout_volume_slider">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="line_playout_volume_text">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_getspeakervolume">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>130</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>播放静音</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_microphone_test">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>麦克风测试</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_peaker_test">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>播音测试</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="pgb_microphone_test">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>140</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="pgb_speaker_test">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>210</y>
      <width>140</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_set_audio_devices">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>设置设备</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbx_ns">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ns(噪声消除)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_get_audio_record">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>采集设备</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_get_recording_devices">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_get_playout_devices">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>290</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_get_audio_playout">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>播放设备</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_get_audio_devices">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>40</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>重获设备</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_get_recording_devices">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>240</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>获取当前采集设备</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_get_playout_devices">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>280</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>获取当前播放设备</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_get_video_device">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>视频设备</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_get_video_device">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>330</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_get_video_device">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>320</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>获取当前视频设备</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbx_echo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>echo(回音消除)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbx_agc">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>agc(自动增益)</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
